Gemfile

```diff
@@ -221,3 +220,6 @@ gem "maintenance_tasks", "~> 2.3"
 gem "stackprof"
 gem "sentry-ruby"
 gem "sentry-rails"
+# for SAML based SSO
+gem 'devise_saml_authenticatable', '~> 1.9.1'
+gem 'ruby-saml', '~> 1.16.0'
```

Comment:
You don't need to add this gem explicitly as it is dependency of devise_saml_authenticatable